test/contracts/deployer-child/elrond.json

```diff
@@ -0,0 +1,6 @@
+{
+    "language": "clang",
+    "source_files": [
+        "/home/bogdan/Elrond/arwen-wasm-vm/test/contracts/deployer-child/deployer-child.c"
```

Comment:
source file is wrong, like path should be not under /home/bogdan

Reply:
hmmm is possible that the build tool put it here